com.unity.netcode.gameobjects/Runtime/Timing/NetworkTime.cs

```diff
@@ -63,7 +63,7 @@ public NetworkTime(int tickRate)
             Assert.IsTrue(tickRate > 0, "Tickrate must be a positive value.");
 
             m_TickRate = tickRate;
-            m_TickInterval = 1f / m_TickRate; // potential floating point precision issue, could result in different interval on different machines
+            m_TickInterval = 1d / m_TickRate; // potential floating point precision issue, could result in different interval on different machines
```

Comment:
I assume this is coming from some other pending PR? Seems a bit odd to lump it into this change, especially if it's an actual bug fix.